pkg/api/internal/testing.go

```diff
@@ -143,6 +143,20 @@ func GetAccounts(handler http.Handler, query url.Values) *httptest.ResponseRecor
 	return rec
 }
 
+func GetBalances(handler http.Handler, query url.Values) *httptest.ResponseRecorder {
```

Comment:
Since @antoinegelloz changed all the functions to use fmt.Sprintf(), you should do the same.